library/src/main/java/com/novoda/downloadmanager/lib/DownloadsDestination.java

```diff
@@ -0,0 +1,63 @@
+package com.novoda.downloadmanager.lib;
+
+final class DownloadsDestination {
```

Comment:
A contract class is just a final class with constants as https://android.googlesource.com/platform/frameworks/base/+/refs/heads/master/core/java/android/provider/ContactsContract.java, that might or might not contain inner classes

I guess that what you mean is to create a DownloadsContract.Batches, DownloadsContract.Status etc?

If you could elaborate a bit more... :)